.gitignore

```diff
@@ -1,3 +1,14 @@
+*.iml
```

Comment:
Could you please explain a bit, what are the downside that you see in having it here, that are worth removing it?